src/ralph/data_importer/management/commands/importer.py

```diff
@@ -79,8 +89,52 @@ def add_arguments(self, parser):
             default='utf-8',
             help="Output encoding",
         )
+        parser.add_argument(
```

Comment:
dir and zipfile are no longer necessary. Use single, positional param for file source path.